Week-1/Exercise1.2/readme.md

```diff
@@ -0,0 +1,4 @@
+### Solution Links
+
+- Solution URL: [Git](https://github.com/jasham/pesto_assignment_website1)
```

Comment:
@jasham Please commit the code as week 1 second assignment. Don't have it under your github repo, have it as part of this pesto repo

Reply:
@jasham you have done it with next.js react way. Can you just do it with simple html, css and js ?

Reply:
But it looks good to me

Reply:
@RajeshSivanesan Thanks for the comments. I can do it in pure HTML, CSS and JS. Shall I do it again ??